source/docs/api-reference/api-usage/status-signals.rst

```diff
@@ -180,7 +179,29 @@ The following signals are time-synchronized:
 ``SignalMeasurement``
 ---------------------
 
-All ``StatusSignalValue`` objects have a ``getDataCopy()`` method that returns a new ``SignalMeasurement`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/StatusSignalValue.SignalMeasurement.html>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/structctre_1_1phoenixpro_1_1_signal_measurement.html>`__) object.
-``SignalMeasurement`` is a `Passive Data Structure <https://en.wikipedia.org/wiki/Passive_data_structure>`__ that provides all the information about a signal at the time of the ``getDataCopy()`` call, which can be useful for data logging.
+All ``StatusSignalValue`` objects have a ``getDataCopy()`` method that returns a new ``SignalMeasurement`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/StatusSignalValue.SignalMeasurement.html>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/structctre_1_1phoenixpro_1_1_signal_measurement.html>`__) object. ``SignalMeasurement`` is a `Passive Data Structure <https://en.wikipedia.org/wiki/Passive_data_structure>`__ that provides all the information about a signal at the time of the ``getDataCopy()`` call, which can be useful for data logging.
 
 .. warning:: ``getDataCopy()`` returns a **new** ``SignalMeasurement`` object every call. **Java** users should **avoid** using this API in RAM-constrained applications.
+
+Latency Compensation
+--------------------
+
+Phoenix Pro provides built-in support for latency compensation via a static function ``getLatencyCompensation()`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/BaseStatusSignalValue.html#getLatencyCompensatedValue(com.ctre.phoenixpro.StatusSignalValue,com.ctre.phoenixpro.StatusSignalValue)>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1_base_status_signal_value.html#a22f020db5abbf556ac7605024309bb26>`__).
+
+.. important:: ``getLatencyCompensatedValue()`` does not automatically refresh the signals. As a result, the user must ensure the ``signal`` and ``signalSlope`` parameters are refreshed before retrieving a compensated value.
+
+.. tab-set::
+
+   .. tab-item:: Java
+      :sync: Java
+
+      .. code-block:: java
+
+         double compensatedTurns = BaseStatusSignalValue.getLatencyCompensatedValue(m_motor.getPosition(), m_motor.getVelocity());
+
+   .. tab-item:: C++
+      :sync: C++
+
+      .. code-block:: cpp
+
+         auto compensatedTurns = BaseStatusSignalValue::GetLatencyCompensatedValue(m_motor.GetPosition(), m_motor.GetVelocity());
```

Comment:
Move this section above the SignalMeasurement section.

Reply:
I think it makes more sense for this to be at the bottom since it's a static function and users should use `BaseStatusSignalValue` over `StatusSignalValue` (at least for C++, otherwise you have to be explicit with the template type).

Otherwise, I'd agree with you.

Reply:
> users should use BaseStatusSignalValue over StatusSignalValue

No they shouldn't, you can't use `BaseStatusSignalValue` as a value type, only through a heap-allocated pointer.

Anyways that's irrelevant to moving this section to above the `SignalMeasurement` section, which is about an API that returns a PDS object that is mostly used for logging data in C++.

Reply:
It can be jarring (again, for C++) to move from normal getters and functions to static function on a different class.

Reply:
Normally I would agree, but we already have the timesync section above SignalMeasurement, and we don't really want the SignalMeasurement section to be all that prominent given that it covers a very small use case.